src/Command/AnalyzeCommand.php

```diff
@@ -133,19 +133,18 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         }
 
         $errorFormat = $input->getOption('error-format');
-        \assert(\is_string($errorFormat) || null === $errorFormat);
+        \assert(\is_string($errorFormat));
 
-        if (null === $errorFormat) {
+        if ('detect' === $errorFormat) {
             $ciDetector = new CiDetector();
-            $errorFormat = 'console';
 
             try {
                 $ci = $ciDetector->detect();
                 if (CiDetector::CI_GITHUB_ACTIONS === $ci->getCiName()) {
                     $errorFormat = 'github';
                 }
             } catch (CiNotDetectedException $e) {
-                // pass and use default
+                $errorFormat = 'console';
```

Comment:
WDYT @smoench ?

Reply:
Wouldn't work in an other CI environment. The CiDetector can detect more than github actions ;-)

Reply:
Makes sense